lib/src/controller.dart

```diff
@@ -13,6 +13,15 @@ class WizardController extends ChangeNotifier {
   final Map<String, WizardRoute> routes;
   late final FlowController<List<WizardRouteSettings>> _flowController;
 
+  bool _isBusy = false;
```

Comment:
in theory, there could be overlapping push/pop operations. it might be good to model this internally as a number, like a reference count and return true if it's > 0